pom.xml

```diff
@@ -263,6 +267,21 @@
       <artifactId>org.eclipse.persistence.moxy</artifactId>
       <version>2.5.2</version>
     </dependency>
+    <dependency>
```

Comment:
@kai-chi why do we need these?

Reply:
@tkakantousis, removed this part and added exclusions